packages/@internationalized/date/docs/ZonedDateTime.mdx

```diff
@@ -61,6 +61,19 @@ let date = parseAbsolute('2021-11-07T07:45:00Z', 'America/Los_Angeles');
 let date = parseAbsoluteToLocal('2021-11-07T07:45:00Z');
 ```
 
+You can also create a `ZonedDateTime` by using `Date` object or epoch time (milliseconds) using one of the following functions:
+
+* <TypeLink links={docs.links} type={docs.exports.fromDate} /> – This function uses a `Date` object with a time zone identifier, e.g. `America/Los_Angeles`, must be provided and the result will be converted into that time zone.
+* <TypeLink links={docs.links} type={docs.exports.fromAbsolute} /> – This function uses an Unix epoch (milliseconds since 1970) , e.g. `1688023843144` with a time zone identifier, e.g. `America/Los_Angeles`, must be provided and the result will be converted into that time zone.
```

Comment:
Small suggestion to match the doc style in previous
```suggestion
* <TypeLink links={docs.links} type={docs.exports.fromDate} /> – This function creates a `ZonedDateTime` from a `Date` object. A time zone identifier, e.g. `America/Los_Angeles`, must be passed, and the result will be converted into that time zone.
* <TypeLink links={docs.links} type={docs.exports.fromAbsolute} /> – This function creates a `ZonedDateTime` from an Unix epoch (e.g. `1688023843144`, representing milliseconds since 1970). A  time zone identifier, e.g. `America/Los_Angeles`, must be provided and the result will be converted into that time zone.
```